civictechprojects/static/css/MainFooter.css

```diff
@@ -19,7 +19,7 @@
 }
 
 .MainFooter-footer-link::after {
```

Comment:
If we're not using dots to separate items, this could be refactored to remove the ::after psuedoselector.

Probably something like 
```css
.MainFoooter-footer-link {
  padding: 0px 15px;
}
```
and then delete the entry for `.MainFooter-footer-link:last-of-type::after entirely. Also I'd suggest 15/15 left and right instead of 0/30 so the clicktargets are evenly balanced.